client/my-sites/stats/mini-carousel/index.jsx

```diff
@@ -57,6 +73,23 @@ const MiniCarousel = ( { slug, isOdysseyStats } ) => {
 
 	const blocks = [];
 
+	if ( showWriteAPostBanner ) {
+		blocks.push(
+			<MiniCarouselBlock
```

Comment:
It seems it's not caused by this task, but I see we're firing [calypso_stats_traffic_blaze_banner_view](https://mc.a8c.com/tracks/live/?eventname=calypso_stats_traffic_blaze_banner_view) on page load even when this banner is shown as first. And if do click to see the second banner (Blaze) we get another event call.

Are you getting the same result on your side?

Reply:
This could be related to the banner on the bottom, when we remove it, it will be fixed

Reply:
O tried removing that banner and still had the issue, can you confirm it also happens on your side?

Reply:
Testing in a couple of minutes ;)

Reply:
Confirmed not happening. I remove the bottom banner completely for my tests and it works fine!